src/lib/matchers.ts

```diff
@@ -0,0 +1,21 @@
+/**
+ * Collection of useful custom jasmine matchers for tests.
+ */
+export const customMatchers: jasmine.CustomMatcherFactories = {
```

Comment:
I think that, as the build is set up now, this would end up in the released package.

@DevVersion?